<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerbConjugation</class>
 <widget class="QDialog" name="VerbConjugation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Verb Conjugation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>English Verb Conjugation for </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="firstLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>First Person -- I</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="secondLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Second Person -- You</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="thirdLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Third Person -- He She It</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="firstpLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>First Person Plural -- We</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="thirdpLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Third Person Plural -- They</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBttn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="LineEditRU" name="thirdpEditru">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>490</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="secondLblru">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Second Person -- Ты</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="thirdpLblru">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Third Person Plural -- Они</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="firstLblru">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>First Person -- Я</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="thirdLblru">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Third Person -- Он Она Оно</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="firstpLblru">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>First Person Plural -- Мы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="secondEditru">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="firstpEditru">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="thirdEditru">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="firstEditru">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLblru">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>780</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Verb Conjugation for </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="secondpLblru">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Second Person Plural -- Вы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="secondpEditru">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="conjBttn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Conjugations</string>
   </property>
  </widget>
  <widget class="QLabel" name="stemLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Verb Stem</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="stemEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="caseLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Verb Case</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="caseBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>330</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="LineEditEN" name="firstEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="secondEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="thirdEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="firstpEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="thirdpEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="ruVerbEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ruVerbLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Verb</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="enVerbLbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>English Verb</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="enVerbEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="recBttn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Recalculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="recLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Erases all saved&lt;br&gt;verbal values&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="selfCheck">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Oneself</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditRU</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineeditru</header>
  </customwidget>
  <customwidget>
   <class>LineEditEN</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineediten</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>selfCheck</tabstop>
  <tabstop>enVerbEdit</tabstop>
  <tabstop>firstEdit</tabstop>
  <tabstop>secondEdit</tabstop>
  <tabstop>thirdEdit</tabstop>
  <tabstop>firstpEdit</tabstop>
  <tabstop>thirdpEdit</tabstop>
  <tabstop>recBttn</tabstop>
  <tabstop>ruVerbEdit</tabstop>
  <tabstop>caseBox</tabstop>
  <tabstop>stemEdit</tabstop>
  <tabstop>firstEditru</tabstop>
  <tabstop>secondEditru</tabstop>
  <tabstop>thirdEditru</tabstop>
  <tabstop>firstpEditru</tabstop>
  <tabstop>secondpEditru</tabstop>
  <tabstop>thirdpEditru</tabstop>
  <tabstop>cancelBttn</tabstop>
  <tabstop>conjBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
